calico/datamodel_v1.py

```diff
@@ -82,11 +88,22 @@ def dir_for_per_host_config(hostname):
     return dir_for_host(hostname) + "/config"
 
 
+def dir_for_status(hostname):
+    return STATUS_DIR + "/%s" % hostname
+
+
+def key_for_status(hostname):
+    return dir_for_status(hostname) + "/last_reported_status"
+
+
+def key_for_uptime(hostname):
+    return dir_for_status(hostname) + "/uptime"
+
+
 def key_for_endpoint(host, orchestrator, workload_id, endpoint_id):
     return (HOST_DIR + "/%s/workload/%s/%s/endpoint/%s" %
             (host, orchestrator, workload_id, endpoint_id))
 
-
```

Comment:
Did you mean to delete this blank line? Not consistent with spacing in the rest of this file.